.gitlab/hpsf-gitlab-ci.yml

```diff
@@ -7,3 +7,15 @@ test:
     - cmake --build build -j48
     - cd build
     - ctest -V
+
+INTEL-DATA-CENTER-MAX-1100:
+  stage: test
+  tags: [intel-data-center-max-1100]
+  image: intel/oneapi-basekit:2024.2.1-0-devel-ubuntu22.04
+  script:
+    - sycl-ls
+    - export ONEAPI_DEVICE_SELECTOR=level_zero:gpu
+    - cmake -B build -DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_COMPILER=icpx -DKokkos_ENABLE_SYCL=ON -DKokkos_ARCH_INTEL_PVC=ON -DKokkos_ENABLE_TESTS=ON -DCMAKE_CXX_FLAGS="-fsycl-device-code-split=per_kernel -fp-model=precise"
+    - cmake --build build -j48
+    - cd build
+    - ctest -V
```

Comment:
Fine but looking at https://gitlab.spack.io/kokkos/kokkos/-/jobs/12916716 we got ~4500L of output so we might want to use `--output-on-failure`?
Also there is a `--test-dir` option that would avoid the need for changing directory.

Reply:
I'm not sure if having that amount of output is a problem. I would rather reduce it later (to have more information about the tests that ran) if it turns out to be a problem.

Reply:
Sure